### DISCUSSION TITLE
v3 workflow API includes `event` field but v4 API `WorkflowRun` does not

### DISCUSSION THREAD

**Discussion by chdsbd** [community/community#37500] [Apps, API and Webhooks]
The [v3 API for retrieving workflow runs](https://docs.github.com/en/rest/actions/workflow-runs#get-a-workflow-run) includes the `event` that caused the workflow run in the API response. The [v4 API does not include this field](https://docs.github.com/en/graphql/reference/objects#workflowrun).

Without this field in the v4 API, it's not possible to deduplicate check runs to match the GitHub UI.

It seems like this should be a simple field to add to the GraphQL API. Otherwise calls will need to make separate API calls to the v3 REST API to get this information.

**Comment by ossanna16** [comment]
Hi there @chdsbd and welcome to our community! Thank you for the feedback 🙂
